commands/playerdata2.js

```diff
@@ -0,0 +1,192 @@
+                            return 0;
+                        } else {
+                            try {
+                                return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+                            } catch(e) {
+                                return 0;
+                            }
+                        }
+                    }
+                    
+                    function profile(msg_2) {
+                        var layout = ['Playtime', 'AfkTime', 'MapsPlayed', 'JoinCount', 'ChatMessages', 'CommandsUsed', 'RocketsLaunched', 'ResearchCompleted', 'MachinesBuilt', 'MachinesRemoved', 'TilesBuilt', 'TilesRemoved', 'TreesDestroyed', 'OreMined', 'ItemsCrafted', 'ItemsPickedUp', 'Kills', 'Deaths', 'DamageDealt', 'DistanceTravelled', 'CapsulesUsed', 'EntityRepaired', 'DeconstructionPlannerUsed', 'MapTagsMade'];
+                        var layout_dict = {
+                            'Playtime':'Play Time',
+                            'AfkTime':'AFK Time',
+                            'MapsPlayed':'Maps Played',
+                            'JoinCount':'Join Count',
+                            'ChatMessages':'Chat Messages',
+                            'CommandsUsed':'Commands',
+                            'RocketsLaunched':'Rockets Launched',
+                            'ResearchCompleted':'Research Completed',
+                            'MachinesBuilt':'Machines Built',
+                            'MachinesRemoved':'Machines Removed',
+                            'TilesBuilt':'Tiles Placed',
+                            'TilesRemoved':'Tiles Removed',
+                            'TreesDestroyed':'Trees Destroyed',
+                            'OreMined':'Ore Mined',
+                            'ItemsCrafted':'Items Crafted',
+                            'ItemsPickedUp':'Items Picked Up',
+                            'Kills':'Kills',
+                            'Deaths':'Deaths',
+                            'DamageDealt':'Damage Dealt',
+                            'DistanceTravelled':'Distance Travelled',
+                            'CapsulesUsed':'Capsules Used',
+                            'EntityRepaired':'Machines Repaired',
+                            'DeconstructionPlannerUsed':'Decon Planner Used',
+                            'MapTagsMade':'Map Tags Created'
+                        };
+                        
+                        var msg_4 = [];
+                    
+                        for (let i = 0; i < layout.length; i++) {
+                            var msg_3 = [];
+                            
+                            if (layout[i] == 'Playtime' || layout[i] == 'AfkTime') {
+                                try {
+                                    var t1 = msg_2[layout[i]];
+                                } catch (e) {
+                                    var t1 = 0;
+                                }
+                                    
```

Comment:
Again use let. Second check with an if if layout[i] exists if it does not set t1 to
Third what do these variable names mean.

Reply:
```suggestion
                                try {
                                    let t1 = msg_2[layout[i]];
                                } catch (e) {
                                    let t1 = 0;
                                }
                                    
```
We need to ask @PHIDIAS0303 for more details on this one too

Reply:
yes ^but the change to let is good